scripts/build_nim.sh

```diff
@@ -64,7 +66,8 @@ nim_needs_rebuilding() {
 		if ! git checkout -q ${NIM_COMMIT} 2>/dev/null; then
 			# Pay the price for a non-default NIM_COMMIT here, by fetching everything.
 			# (This includes upstream branches and tags that might be missing from our fork.)
-			git remote add upstream https://github.com/nim-lang/Nim
+			git remote remove upstream 2>/dev/null || true
+			git remote add upstream "${NIM_COMMIT_REPO_URL}"
```

Comment:
let's add a new remote name for this, if it is set - ie `extra` or something like